tests/Tests/Isolated/Common/Twig/fixtures/render/care-plan-reason-row-populated.html

```
@@ -0,0 +1,47 @@
+<div class="form-row reasonCodeContainer reason_code " id="reason_code_2">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)